src/snowflake/cli/plugins/object/stage/commands.py

```diff
@@ -121,3 +123,27 @@ def stage_diff(
     """
     diff: DiffResult = stage_diff(Path(folder_name), stage_name)
     return ObjectResult(str(diff))
+
+
+@app.command("execute", requires_connection=True)
+def execute(
+    stage_path: str = typer.Argument(
+        None,
+        help="Execute immediate all files from the path. Files can be filtered with glob like pattern, e.g. `@stage/**/*.sql`.",
```

Comment:
```suggestion
        help="Stage path with files to be execute. For example @stage/dev.",
```

Reply:
I wouldn't change the example, at first I assumed that only prefix-filtering is allowed - IMO it's good to have a hint

Reply:
I added `*` to example `For example @stage/dev/*`